### ISSUE TITLE
Persistence 

### ISSUE THREAD

**Issue by travisghansen** [halkeye-helm-charts/mosquitto#2]
Support creating a PVC to work with mosquito persistence options.

**Comment by halkeye** [comment]
Its not something I use, but totally up for a PR

**Comment by travisghansen** [comment]
Yeah, I use persistence to ensure retained messages can be store in the sqlite DB..

**Comment by halkeye** [comment]
Ya totally a legit case. Just not something I use and want to develop. There's plenty of examples out there. Infact octoprint and thelounge come to mind as good examples of my own if you want to implement it

**Comment by travisghansen** [comment]
I already made custom assets not based on the this chart. You can close if you wish. At some later point I may revisit the chart.
